tools/accuracy_checker/accuracy_checker/annotation_converters/README.md

```diff
@@ -502,6 +502,9 @@ The main difference between this converter and `super_resolution` in data organi
 * `vimeo90k` - converts Vimeo-90K dataset for a systematic evaluation of video processing algorithms to `SuperResolutionAnnotation`.
   * `annotation_file` - path to text file with list of dataset setuplets included in test.
   * `add_flow` - allows annotation of flow data (optional, default `False`).
+* `yolo_labeling` - converts object detection dataset with annotation in YOLO labeling format to `DetectionAnnotation`.
+  * `data_dir` - path to directory with annotation files in txt format and images.
```

Comment:
do images and labels are stored in the same directory or separated?

Reply:
yes, data_dir is directory for both images and annotations and they are stored in the same directory